amazon_sqs_send_message_block.py

```diff
@@ -36,23 +36,29 @@ class SQSSendMessage(SQSBase):
     # Should this block include batch sending of messages
 
     def process_signals(self, signals):
+        new_signals = []
         for signal in signals:
             try:
                 self.logger.debug("Sending message via {} queue".format(
                     self.queue_url(signal)))
 
-                self.client.send_message(QueueUrl=self.queue_url(signal),
-                                         DelaySeconds=self.delay_seconds(signal),
-                                         MessageAttributes=self.message_attributes(signal),
-                                         MessageBody=self.message_body(signal))
-
-                # RESPONSE
-                # {
-                #     'MD5OfMessageBody': 'string',
-                #     'MD5OfMessageAttributes': 'string',
-                #     'MessageId': 'string',
-                #     'SequenceNumber': 'string'
-                # }
+                response = self.client.send_message(
+                    QueueUrl=self.queue_url(signal),
+                    DelaySeconds=self.delay_seconds(signal),
+                    MessageAttributes=self.message_attributes(signal),
```

Comment:
I see your comment up above and I'm not sure this is doing what you expect. Put together a unit test where you mock `boto3` and the assert that `send_message` is called with the correct arguments. I think that will help clear up what's happening here without needing to even run the block for real.